processed 8 tasks

task 2 'run'. lines 59-71:
Error: Script execution failed with VMError: {
    major_status: UNKNOWN_INVARIANT_VIOLATION_ERROR,
    sub_status: Some(1),
    location: script,
    indices: [],
    offsets: [(FunctionDefinitionIndex(0), 7)],
}

task 3 'run'. lines 73-86:
Error: Script execution failed with VMError: {
    major_status: UNKNOWN_INVARIANT_VIOLATION_ERROR,
    sub_status: Some(1),
    location: script,
    indices: [],
    offsets: [(FunctionDefinitionIndex(0), 10)],
}

task 6 'run'. lines 116-127:
Error: Script execution failed with VMError: {
    major_status: UNKNOWN_INVARIANT_VIOLATION_ERROR,
    sub_status: Some(1),
    location: script,
    indices: [],
    offsets: [(FunctionDefinitionIndex(0), 8)],
}
